### ISSUE TITLE
Add backfeed support for GitHub Discussions (blocked on API support)

### ISSUE THREAD

**Issue by aarongustafson** [snarfed/bridgy#1077]
GraphQL endpoint: https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions

Using GitHub issues on https://WebWeWant.fyi for individual wants (e.g., https://github.com/WebWeWant/webwewant.fyi/discussions/191).

+ @tantek

**Comment by tantek** [comment]
Most important here is Bridgy backfeed support for existing discussion posts on GitHub, similar to the backfeed support for responses to GitHub issues.

Bridgy Publish of a "discussion" post would be great too, should that be a separate issue / feature request?

(Originally published at: https://tantek.com/2021/292/t2/)

**Comment by aarongustafson** [comment]
I submitted a support request with GitHub and referenced this issue.

Looking at the docs, it seems that it _might_ be possible to infer the URL (if missing) from `resourcePath`
